Source/radiation/Radiation.cpp

```diff
@@ -2148,33 +2202,33 @@ void Radiation::get_rosseland_v_dcf(MultiFab& kappa_r, MultiFab& v, MultiFab& dc
                                  BL_TO_FORTRAN_ANYD(S[mfi]),
                                  igroup, igroup, 1);
 
-            kp.resize(reg);
-#pragma gpu box(reg) sync
-            ca_compute_planck(AMREX_INT_ANYD(reg.loVect()), AMREX_INT_ANYD(reg.hiVect()),
-                              BL_TO_FORTRAN_ANYD(kp),
-                              BL_TO_FORTRAN_ANYD(S[mfi]),
-                              igroup, igroup, 1, 0.0);
+            const Real nu = nugroup[igroup];
 
-            kp2.resize(reg);
-            S[mfi].plus<RunOn::Device>(dT, UTEMP, 1);
-            Gpu::synchronize();
+            amrex::ParallelFor(reg,
```

Comment:
Note: there were two calls to compute_planck here, and in between we were bumping up state(i,j,k,UTEMP) by dT, computing the Planck opacity again, and then resetting the temperature. I combined them into a single parallel loop for simplicity.